walk-safe/package.json

```diff
@@ -4,14 +4,17 @@
   "private": true,
   "dependencies": {
     "@apollo/client": "^3.3.21",
+    "@atomap/use-places-autocomplete": "^1.0.4",
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
     "apollo-server": "^2.25.2",
+    "google-maps-react": "^2.0.6",
```

Comment:
Thx for reinstalling these packages - hopefully that'll get us all on the same page.